kroxylicious-filters/kroxylicious-record-validation/src/main/java/io/kroxylicious/proxy/filter/schema/validation/bytebuf/AllValidBytebufValidator.java

```diff
@@ -16,7 +16,7 @@
 class AllValidBytebufValidator implements BytebufValidator {
 
     @Override
-    public CompletionStage<Result> validate(ByteBuffer buffer, int length, Record record, boolean isKey) {
+    public CompletionStage<Result> validate(ByteBuffer buffer, int length, Record kafkaRecord, boolean isKey) {
```

Comment:
the yellow warning about use of a java keyword `record` as a parameter was bugging me.

Reply:
Its ceaselessly  annoying.

Reply:
We should turn this warning off. It offers no value imo. Just because 'record' has been added as a modifier (but is not a keyword in the Java grammar) does not suddenly make it confusing when it's used as a variable name. Java is going to add more of these modifiers: We're going to get "value classes", where `value` will be a modifier. Applying the same rule means that every variable called `value` will get a warning. In this direction lies madness.

Reply:
> Applying the same rule means that every variable called value will get a warning. In this direction lies madness.

Fair point..